### ISSUE TITLE
Consider changing name

### ISSUE THREAD

**Issue by TingPing** [martinszeltins/who-wants-to-be-a-millionaire#4]
I think its advisable to name the project anything other than "Who wants to be a millionaire" which is an actively used Trademark for real products: https://store.steampowered.com/app/1356240/Who_Wants_To_Be_A_Millionaire/

It would be very easy to convince a lawyer to send you a cease and desist. I'd suggest a more generic name.

**Comment by martinszeltins** [comment]
> I think its advisable to name the project anything other than "Who wants to be a millionaire" which is an actively used Trademark for real products: [store.steampowered.com/app/1356240/Who_Wants_To_Be_A_Millionaire](https://store.steampowered.com/app/1356240/Who_Wants_To_Be_A_Millionaire/)
> 
> It would be very easy to convince a lawyer to send you a cease and desist. I'd suggest a more generic name.

I get what you're saying, this was just the first thing I thought of, I am open to suggestions for the name. Although there are thousands of "Who Wants To Be a Millionaire" games online and I think as long as you're not trying to sell it and make money off of it I don't think anyone will care.

**Comment by TingPing** [comment]
> I think as long as you're not trying to sell it and make money off of it I don't think anyone will care.

This is not how trademark law works.

You could just do a spin on the name like *Who wants to be a zillionare* or something silly.

**Comment by martinszeltins** [comment]
> This is not how trademark law works.

I know but the likelihood of them taking action is very very low.

Maybe it could be called just "Millionaire" nice and short.

**Comment by TingPing** [comment]
Yeah that sounds reasonable.

**Comment by KunalKumar-Code** [comment]
I agree with @TingPing here, you should change the name. How about "Million Dollar Quiz"?
